api/end_device.proto

```diff
@@ -215,18 +219,18 @@ message MACSettings {
 
   // Whether the device supports 32-bit frame counters.
   // If unset, the default value from Network Server configuration will be used.
-  google.protobuf.BoolValue supports_32_bit_f_cnt = 12 [(gogoproto.customname) = "Supports32BitFCnt"];
+  BoolValue supports_32_bit_f_cnt = 12 [(gogoproto.customname) = "Supports32BitFCnt"];
```

Comment:
Doesn't this break API compatibility? A `google.protobuf.BoolValue` renders to JSON as a boolean. A `BoolValue` seems to render to JSON as an object.